### DISCUSSION TITLE
My window becomes inactive after a click outside

### DISCUSSION THREAD

**Discussion by notkojak** [seleniumbase/SeleniumBase#1568] [Q&A] [answered]
Hello again, when I launch my script, everything works perfectly, but as soon as I leave the window, the script stops or rather, does not find any element until I click again on the window.

Do you have any idea about this problem?

Thank you, have a nice day.

**Comment by mdmintz** [comment, marked as answer]
That would depend on what you're trying to do in your script. If you need to hover over an element and then click something, then moving the mouse at the same time could interfere with that. You could try running the script in ``headless`` mode to prevent outside mouse-movement from impacting the script in any way.

**Reply by mdmintz** [reply]
You can add ``--headless`` on the command-line when you call your script with ``python``/``pytest``.
